.travis.yml

```diff
@@ -1,38 +1,37 @@
 language: ruby
-sudo: false
 cache: bundler
 
 rvm:
-  - 2.5.3
-  - 2.4.5
-  - 2.3.8
-  - 2.2.9
-  - 2.1.10
+  - 2.7
+  - 2.6
+  - 2.5
+  - 2.4
+  - 2.3
+  - 2.2
+  - 2.1
   - ruby-head
+  - jruby-9.2.11.1
+  - jruby-head
 
 matrix:
   include:
-    - rvm: 2.5.3
+    - rvm: 2.5
```

Comment:
Do you want these to always run with the latest ruby? In this case, 2.7? Same for the integration tests below.